classes/local/form/preferences_form.php

```diff
@@ -56,7 +56,9 @@ public function define_configured_factors($mform) {
             'factor',
             'devicename',
             'created',
+            'createdfromip',
             'modified',
+            'lastlogon',
```

Comment:
Do we really need modified and lastlogin? In practice won't they end up always being the same?

Reply:
Actually yes, if we revoke a token that we can no longer access, then we modify it but don't update lastlogin